app/core/models.py

```diff
@@ -202,3 +202,17 @@ class Faq(AbstractBaseModel):
 
     def __str__(self):
         return f"{self.question}"
+
+
+class Faq_viewed(AbstractBaseModel):
+    """
+    Faq_viewed has child relation to Faq and tracks how many times an FAQ has been viewed
+    """
+
+    faq = models.ForeignKey(Faq, null=True, editable=False, on_delete=models.CASCADE)
```

Comment:
Editable seems to be a django admin thing and might not be needed. I think the API behavior is controlled by the serializer.

Reply:
Maybe add this? because django admin will show "Faq vieweds" otherwise
```python
class Meta:
    verbose_name = "FAQ view"
```

I thought about changing the model name to FaqView, but then it sounds like something that belongs in `views.py`

Reply:
I made the changes.